pkg/minikube/registry/drvs/docker/docker_test.go

```diff
@@ -135,8 +135,8 @@ func TestCheckDockerVersion(t *testing.T) {
 	}...)
 
 	for _, c := range tc {
-		t.Run("checkDockerVersion test", func(t *testing.T) {
-			s := checkDockerVersion(c.version)
+		t.Run("checkDockerEngineVersion test", func(t *testing.T) {
```

Comment:
will this test be skipped if docker is not in the path?

Reply:
No, but this function doesn't interact with Docker, a version is passed to it and it returns the driver status based on the version.

For example, if the version passed in is older than the minimum required version, return needs update in the driver status.

Reply:
looks good to me